### ISSUE TITLE
fixed last column

### ISSUE THREAD

**Issue by lsentkiewicz** [schrodinger/fixed-data-table-2#121]
```jsx
<Table
    rowHeight={50}
    rowsCount={rows.length}
    width={5000}
    height={5000}
    headerHeight={50}>
    <Column
      header={<Cell>Col 1</Cell>}
      cell={<Cell>Column 1 static content</Cell>}
      width={2000}
      fixed
    />
    <Column
      header={<Cell>Col 2</Cell>}
      cell={<MyCustomCell mySpecialProp="column2" />}
      width={1000}
    />
    <Column
      header={<Cell>Col 3</Cell>}
      cell={({rowIndex, ...props}) => (
        <Cell {...props}>
          Data for column 3: {rows[rowIndex][2]}
        </Cell>
      )}
      width={2000}
      fixed
    />
  </Table>
```

It will be rendered as:
`Col 1 | Col 3 | Col 2`

expected:
`Col 1 | Col 2 | Col 3`

Cols `Col 1` and `Col 3` are fixed.
Is there any way to make the last column as fixed?

**Comment by lsentkiewicz** [comment]
I was able to fix it in my fork https://github.com/lsentkiewicz/fixed-data-table-2

See demo video http://take.ms/5k4WV
The only issue is that there is no shadow on the right separator.

**Comment by KamranAsif** [comment]
Very cool. I think we might want to hold off on this until v1.0+. We are in the middle of a redux migration and this would send us back a bit.

**Comment by ch-masan** [comment]
Cool, I need this

**Comment by dingchaoyan1983** [comment]
is this merged?

**Comment by wcjordan** [comment]
No we still don't support a last fixed column.  For now you could implement this with 2 separate FDT grids with synced row scroll if you wanted to.  If you do, it'd be great if you could submit an example to our docs for it.

**Comment by lsentkiewicz** [comment]
@wcjordan 
I tried to combine two tables. There are also other libraries that allow it https://github.com/bvaughn/react-virtualized
But it doesn't work good on mobile.

**Comment by wcjordan** [comment]
That's too bad.  Our mobile support isn't strongly tested for cases like these.  If you have an idea of what doesn't work well on mobile we might be able to fix that, but don't want to support this outright at this time.

**Comment by calebmeyer** [comment]
@KamranAsif how is the redux migration going? Is this something we can expect to pick back up soon?

**Comment by wcjordan** [comment]
Not sure Kam's available right now, but given it's dragged on and @lsentkiewicz's work looks reasonable, I think we can accept a PR if you could put one together.

**Comment by xhe** [comment]
We also have same requirement, can I know when we can have this feature?
Or @lsentkiewicz, can you send me steps on how how to enable right fixed column? In your fork branch, there are too many updates.
Thanks

**Comment by xhe** [comment]
Hello, May I ask when this improvement can be accepted into master branch? we highly expected this fix, but we also need new features in current fixed data table2, so please help to merge this fix into the main release branch. Thanks

**Comment by wcjordan** [comment]
We would accept a PR for this, but we're unlikely to work on it.

**Comment by xhe** [comment]
any plan in the near future?

**Comment by wcjordan** [comment]
Since we're a small open source project we don't normally develop new features ourselves.  We depend on members of the community to contribute back examples and features.

I think @ardok is working on a way to do this without changes to FDT (see #203).  You could reach out there and see if he could contribute an example back to the repo or walk you through a similar approach.

**Comment by xhe** [comment]
@lsentkiewicz , I like your solution very much for rightFixedColumn. But it depends on old version of  FDT, now, FDT is already 0.8, is that possible for you to put your solution into FDT 0.8? Thanks very much.

**Comment by lsentkiewicz** [comment]
@xhe 
Sorry, no time now.
I will be updating FDT to the latest version, but it will be in 1-2 months.

**Comment by xhe** [comment]
Thanks for your reply.
Will be waiting for your update sooner.
Thanks

**Comment by xhe** [comment]
@lsentkiewicz , can you tell me how you generated the content within directory 'internal'? I can not find out how to do it. is that generated automatically with some command?

**Comment by ardok** [comment]
This is how I did mine:
https://gist.github.com/ardok/4aa9500f5dae3b462427b7cd54404176

**Comment by xhe** [comment]
Hi Ardo,
Can you tell me how you  generated the folder content under 'internal'?
Without these, the component won't work.
I don't know how to do that.
Thanks

**Comment by garytang8** [comment]
I opened a PR for this and it'd be great to get it reviewed, merged in and released. @wcjordan or @KamranAsif , can you take a look at it?

**Comment by wcjordan** [comment]
Fixed in release 0.8.4
